### DISCUSSION TITLE
Nice mention today on a popular AI/Stable Diffusion tutorial channel:

### DISCUSSION THREAD

**Discussion by CCpt5** [Sanster/IOPaint#158] [General]
Just wanted to mention that your app was mentioned today in a new video on Youtube here:  **https://www.youtube.com/watch?v=aYia7Jvbjno**  
  
I've enjoyed it a lot the past two months since I found it, so I wanted to make sure you noticed the mention.

Thanks for everything!!

-J

**Comment by Sanster** [comment]
Yes! The video he made is much better than my 30-second demo video, and I've added it to the readme
